example/ios/.xcode.env.local

```diff
@@ -0,0 +1 @@
+export NODE_BINARY=/Users/cristiano.tofani/.nvm/versions/node/v18.19.0/bin/node
```

Comment:
This file should not be committed and added to the `.gitignore` file.

Reply:
[7a98393](https://github.com/pagopa/io-react-native-crypto/pull/32/commits/7a98393246d7c52fd893d98970214f01d551dec3) to fix